examples/host/simpleResolver/simpleResolver.py

```diff
@@ -153,13 +189,20 @@ def main():
     trait_set = set(args.traitset.split(","))
 
     # Create an OpenAssetIO context that describes the calling
-    # environment
+    # environment. The lifetime of this object is very important in
+    # OpenAssetIO. It is used to tie together related calls into the API
+    # so that the manager can (if appropriate) ensure temporal coherency
+    # across resolves.
```

Comment:
[Question] Is there a way to simplify the phrase "temporal coherency."? It almost seems like a technical phrase that could be served by a link to a definition.

Reply:
Howes [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/839/commits/e0a84a059fc9d0408560ae610efc98c787f7d37e)?